src/Symfony/Component/Routing/Attribute/AsRoutingService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class AsRoutingService
```

Comment:
All code being in the `FrameworkBundle`, it would reduce cross-component dependencies if this classe was there too. I'm not sure this class as any value when the routing component is used alone.

Reply:
true. this attribute does not do anything inside the routing component